### PR TITLE
Return empty handler if rgb doesn't exist when trying to add alpha value

### PR DESCRIPTION
**Pull request by larshg**
Print error and return empty callback if only alpha channel is listed in the header.

Should fix:
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=50652

### CODE REVIEW

**Comment by larshg** [comment]
I changed it to use the method you mentioned - thanks :+1: 

And I changed to just emit an error as well as returning an empty callback, as it seems to be an ill-defined header anyways, for  a PLY file. At least a usable ply file with respect to PCL.

File: io/src/ply_io.cpp [hunk 1]
```diff
@@ -213,6 +214,12 @@ namespace pcl
       }
       if (property_name == "alpha")
       {
+        if(pcl::getFieldIndex(*cloud_,"rgb") == -1)
```
Comment (mvieth) [inline io/src/ply_io.cpp:217]: How about adding a `bool` return value to `amendProperty` indicating success/failure? That would avoid calling `getFieldIndex` twice

File: common/src/PCLPointCloud2.cpp [hunk 2]
```diff
@@ -39,6 +39,7 @@
  */
 
 #include <vector>
+#include <iterator>
```
Comment (mvieth) [inline common/src/PCLPointCloud2.cpp:42]: Let's not add this include if not needed

**Review by mvieth — APPROVED** [review]
Looks good :+1:

**Pull request merged**